client/src/components/Tickets/TicketModal.js

```diff
@@ -170,6 +190,11 @@ const TicketHistoryCard = ({
       </CardHeader>
       <Collapse isOpen={isOpen}>
         <CardBody>{content}</CardBody>
+        {attachmentTitles && (
```

Comment:
This actully still shows the `Attachments` component. We can instead look at `.length` and cast the result to a boolean:
```suggestion
        {!!attachmentTitles.length && (
```